2024/call-for-papers.md

```diff
@@ -64,14 +64,14 @@ Please adhere to the submission deadlines and other key dates outlined below. No
 - September 24, 2024 : Paper submission deadline.
 - October 08, 2024 : Acceptance Notification
 - December 08, 2024 : Workshop & Mentoring event
-  All submission deadlines are **23:59 Pacific Time (PT)**.
+
+All submission deadlines are **23:59 Pacific Time (PT)**.
 
 **Attendance Awards**
 
-<!-- We are pleased to offer travel grants to a select number of authors with accepted papers. The exact amount of each grant will be determined based on the number of applications received and will be disclosed following the paper acceptance notifications. These grants are intended to help offset travel expenses related to attending the workshop, including flights, accommodations, and conference registration. Please note that the grant may not cover all incurred expenses. Recipients will be required to submit receipts for travel expenses to receive their award.
-Priority for travel grants will be given to authors from **developing countries**. However, we regret that we cannot guarantee grants for all accepted papers. The final grant amounts will depend on the volume of applications and will be communicated after the acceptance notifications. -->
+We are pleased to offer travel grants to a select number of authors with accepted papers. The exact amount of each grant will be determined based on the number of applications received and will be disclosed following the paper acceptance notifications. These grants are intended to help offset travel expenses related to attending the workshop, including flights, accommodations, and conference registration. Please note that the grant may not cover all incurred expenses. Recipients will be required to submit receipts for travel expenses to receive their award.
```

Comment:
@williem-scientist I think let's not put `accommodations` here since we won't likely be able to cover accommodations for participants. This might change depending on how much we can get from sponsors. 
Once we have a better financial number, we can decide whether to support accommodations. For now, let's remove that.

Thank you very much for your prompt response.

Reply:
I have removed the `accommodations`. It remains `flights` and `conference registration`.